app/helpers/era_av_helper.rb

```diff
@@ -0,0 +1,29 @@
+# Copyright 2017, University of Alberta Libraries.
```

Comment:
Yeah, dunno. I find it a bit annoying, but I guess if we ever push code upstream then we should follow the upstream conventions. (Not that this file would every get pushed upstream).